manifest.yaml

```diff
@@ -1,13 +1,17 @@
 id: lnd
 title: Lightning Network Daemon
-version: 0.14.2.4
-release-notes: Fix migrations to properly handle the bitcoin configuration for updates from 0.13.x versions and ensure any version will successfully migrate to the latest version going forward
+version: 0.15.0
+release-notes: |-
+  * New LND v0.15.0-beta: [Upstream Release Notes](https://github.com/lightningnetwork/lnd/blob/master/docs/release-notes/release-notes-0.15.0.md)  
+  * Add "Allow Circular Route" Config Option
+  * Enable Watchtowers 
 license: mit
 wrapper-repo: "https://github.com/Start9Labs/lnd-wrapper"
 upstream-repo: "https://github.com/lightningnetwork/lnd"
 support-site: "https://lightning.engineering/slack.html"
 marketing-site: "https://lightning.engineering/"
 build: ["make"]
+eos-version: 0.3.1.1
```

Comment:
We should not define this override unless there's a good reason

Reply:
well I thought that if you try to install LND with JS migrations on a EOS <v0.3.1.1, the migration path won't work. is this not true?

Reply:
Yes, but as long as you pack with sdk v0.3.1.1, that will be implicit in the s9pk. We'll have to keep updating this in the manifest if we define it here, whereas it will always stay in sync with your sdk if now.

Reply:
ah ok, I'll remove it

Reply:
run `embassy-sdk inspect manifest lnd.s9pk` to see what version it was packed with.

Reply:
```
{
  "eos-version": "0.3.1.1",
  "id": "lnd",
  "title": "Lightning Network Daemon",
  "version": "0.15.0",
```

Looking good to me. manifest line removed.